tweakable_encryptor_test.go

```diff
@@ -6,6 +6,7 @@ package saltpack
 import (
 	"bytes"
 	"crypto/sha512"
+	"fmt"
```

Comment:
Since the tweakable encryptor is only used for unit tests, it seems possible to me to:

- modify the real encryptor to have a `testDelegate` interface with all the tweak functions;
- have a non-exported function that constructs the encryptor with a passed-in `testDelegate`;
- make the exported constructor call the non-exported one with a `nil` or do-nothing `testDelegate`.

Anyway, I don't think we have to do this now, but something to think about...